docs/reference_guide.md

```diff
@@ -2701,6 +2702,25 @@ contains the memcopy'd value so the value at `print()` invocation time will be
 printed.  However for maps, only the handle to the map is queued up, so the
 printed map may be different than the map at `print()` invocation.
 
+# 24. `strftime()`: Formatted timestamp
+
+Syntax:
+- `strftime(const char *format, int nsecs)`
+
+This returns a formatted timestamp that is printable with `printf`. The format string must be supported by `strftime(3)`. Use format specifier "%s" when printing the return value. Note that `strftime` does not actually return a string.
```

Comment:
can you align this to the rest of the doc, 80 (or 100?) char lines

Reply:
sure